apps/frontend/src/pages/project/libs/components/contributors-list/contributors-list.tsx

```diff
@@ -13,12 +14,13 @@ const ContributorsList = ({
 	contributors,
 	isLoading,
 }: Properties): JSX.Element => {
+	const hasContributors = contributors.length > EMPTY_LENGTH;
+
 	return (
 		<div className={styles["container"]}>
 			<h2 className={styles["title"]}>Contributors</h2>
-			{isLoading ? (
-				<Loader />
-			) : (
+			{isLoading && <Loader />}
+			{!isLoading && hasContributors && (
```

Comment:
let's move these conditions to separate constants - isListShown and isEmptyPlaceholderShown
```suggestion
			{isListShown && (
```